### ISSUE TITLE
has_one serialization fails for nil association 

### ISSUE THREAD

**Issue by elrosa** [Netflix/fast_jsonapi#35]
Attempt to serialize model with `has_one` association fails if association is nil:

**`NoMethodError (undefined method 'id' for nil:NilClass)`**

`(fast_jsonapi-1.0.16/lib/extensions/has_one.rb:12)`

If associated model exists, it gets serialized correctly.

Example:
```Ruby
class User < ApplicationRecord
  has_one :profile
end
 
class Profile < ApplicationRecord
   belongs_to :user
end

class UserSerializer
  include FastJsonapi::ObjectSerializer
  attributes :id, :name
  has_one :profile
end

UserSerializer.new(User.create(name: "John")).serialized_json
NoMethodError (undefined method 'id' for nil:NilClass)

```

**Comment by grossadamm** [comment]
Great find! We'll get a test to replicate and fix.

**Comment by grossadamm** [comment]
Looks like @jeremyjung is on this. Thanks!

**Comment by christophersansone** [comment]
@shishirmk We have a merged PR... should this issue be closed?

**Comment by shishirmk** [comment]
Fixed in Version 1.1.0
